### PR TITLE
qtile-top: fix IPC client creation

### LINKED ISSUE #1656: qtile-top crash 'Client' object has no attribute 'tracemalloc_dump'
# Issue description

Tried to run `qtile-top` and it crashed. I'm not sure should be done before that script  - no documentation.
Tried 
```
 arnas@zordmachine  ~  qtile-cmd -o cmd -f tracemalloc_dump
[False, 'Trace not started']
 arnas@zordmachine  ~  qtile-cmd -o cmd -f tracemalloc_toggle
 arnas@zordmachine  ~  qtile-cmd -o cmd -f tracemalloc_dump
[True, '/home/arnas/.cache/qtile/qtile_tracemalloc.dump']
```
but this didn't helped

# Qtile version

5bd8e57f78b72ed9de4d10bf51af6db51561d794 (latest git version)

# Stack traces

Traceback (most recent call last):
  File "/usr/bin/qtile-top", line 11, in <module>
    load_entry_point('qtile==0.15.0', 'console_scripts', 'qtile-top')()
  File "/usr/lib/python3.8/site-packages/libqtile/scripts/qtile_top.py", line 164, in main
    curses.wrapper(get_stats, client, limit=lines, seconds=seconds,
  File "/usr/lib/python3.8/curses/__init__.py", line 105, in wrapper
    return func(stdscr, *args, **kwds)
  File "/usr/lib/python3.8/site-packages/libqtile/scripts/qtile_top.py", line 91, in get_stats
    snapshot = get_trace(client, force_start)
  File "/usr/lib/python3.8/site-packages/libqtile/scripts/qtile_top.py", line 63, in get_trace
    (started, path) = client.tracemalloc_dump()
AttributeError: 'Client' object has no attribute 'tracemalloc_dump'

# Configuration

https://github.com/zordsdavini/qtile-config
Arch

### PR DESCRIPTION
**Pull request by tych0**
Not sure if this fixes #1656, but looking at the tests it seems like it should?

### CODE REVIEW

**Comment by zordsdavini** [comment]
I can confirm - this fixes issue :+1:

File: libqtile/scripts/qtile_top.py [hunk 1]
```diff
@@ -158,6 +158,8 @@ def main():
     else:
         socket = opts.socket
     client = ipc.Client(socket)
+    client = command_interface.IPCCommandInterface(client)
+    client = command_client.InteractiveCommandClient(client)
```
Comment (ramnes) [inline libqtile/scripts/qtile_top.py:162]: Can't you replace all these lines with just `client = command_client.CommandClient()`?

IIRC @pawamoy did something for that recently.

Comment (pawamoy) [reply libqtile/scripts/qtile_top.py:162]: I did in #1635, however `client = command_client.CommandClient()` will always use `find_sockfile`, which might not be wanted here.

Comment (tych0) [reply libqtile/scripts/qtile_top.py:162]: I think we should pester @flacjacket to write down how all this works :)

Comment (flacjacket) [reply libqtile/scripts/qtile_top.py:162]: That is a good idea :) I'll see about updating the docs. Better names for these may also be helpful, I can think about it as I write it up, but would be open if anyone has any suggestions.

This change looks good!

**Pull request merged**